.travis.yml

```diff
@@ -32,6 +32,14 @@ matrix:
     env: CI_CATEGORY=UNIT_TESTS
     jdk: oraclejdk9
     scala: 2.12.6
+  - os: linux
```

Comment:
Those are not tested on purpose (2.10 support will be removed, 2.11/java9 built for integration test, ..., test matrix is already quite large)

Reply:
want me to remove 2.10 support from the build?

Reply:
not to re-add it to the CI matrix

Reply:
you want me to revert all the changes on this file, and you will remove the support for 2.10 in another PR?